patches/minecraft/net/minecraft/client/renderer/BlockModelShapes.java.patch

```diff
@@ -1,6 +1,16 @@
 --- ../src-base/minecraft/net/minecraft/client/renderer/BlockModelShapes.java
 +++ ../src-work/minecraft/net/minecraft/client/renderer/BlockModelShapes.java
-@@ -391,5 +391,6 @@
+@@ -206,6 +206,9 @@
+ 
+     public IBakedModel func_178125_b(IBlockState p_178125_1_)
+     {
++        if (p_178125_1_ instanceof net.minecraftforge.common.property.IExtendedBlockState)
```

Comment:
Ya, this just looks like a side effect of your ordering of things below, why even introduce the concept of sending in extended states to this function?